lib/serum/dev_server/command_handler/impl.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Serum.DevServer.CommandHandler.Impl do
+  @moduledoc false
+
+  _moduledocp = """
+  A implementation of `Serum.DevServer.CommandHandler` behaviour.
+  """
+
+  alias Serum.DevServer.CommandHandler
+
+  @behaviour CommandHandler
+
+  @impl CommandHandler
+  @spec open_url(url :: binary()) :: {:ok | :error}
+  def open_url(url) do
+    open_command =
+      case :os.type() do
+        {:unix, :darwin} -> "open"
+        {:unix, _} -> "xdg-open"
+        {:win32, _} -> "start"
```

Comment:
I cannot verify it right now, but this shouldn't work because `start` is not an external program which can be located using the `%PATH%` environment variable, but it's a built-in shell command of `cmd.exe`. Have you tested this code in Windows? 👀

Reply:
You are right. I found the way to open a browser in windows with `rundll32`.